dropwizard-app/src/test/java/org/coner/resource/HandicapGroupsResourceTest.java

```diff
@@ -0,0 +1,132 @@
+package org.coner.resource;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dropwizard.jackson.Jackson;
+import io.dropwizard.jersey.validation.ConstraintViolationExceptionMapper;
+import io.dropwizard.testing.FixtureHelpers;
+import io.dropwizard.testing.junit.ResourceTestRule;
+import org.coner.api.response.ErrorsResponse;
+import org.coner.boundary.HandicapGroupBoundary;
+import org.coner.core.ConerCoreService;
+import org.coner.core.domain.HandicapGroup;
+import org.coner.util.JacksonUtil;
+import org.eclipse.jetty.http.HttpStatus;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.math.BigDecimal;
+
+import static org.fest.assertions.Assertions.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+/**
+ *
+ */
+public class HandicapGroupsResourceTest {
+    private final HandicapGroupBoundary handicapGroupBoundary = mock(HandicapGroupBoundary.class);
+    private final ConerCoreService conerCoreService = mock(ConerCoreService.class);
+
+    private ObjectMapper objectMapper;
+
+    @Rule
+    public final ResourceTestRule resources = ResourceTestRule.builder()
+            .addResource(new HandicapGroupsResource(handicapGroupBoundary, conerCoreService))
+            .addProvider(new ConstraintViolationExceptionMapper())
+            .build();
+
+    @Before
+    public void setup() {
+        reset(handicapGroupBoundary, conerCoreService);
+
+        objectMapper = Jackson.newObjectMapper();
+        JacksonUtil.configureObjectMapper(objectMapper);
+    }
+
+    @Test
+    public void whenAddValidHandicapGroupItShouldAddHandicapGroup() throws Exception {
+        org.coner.api.entity.HandicapGroup requestHandicapGroup = objectMapper.readValue(
+                FixtureHelpers.fixture("fixtures/api/entity/handicap_group_add-request.json"),
+                org.coner.api.entity.HandicapGroup.class
+        );
+
+        Entity<org.coner.api.entity.HandicapGroup> requestEntity = Entity.json(requestHandicapGroup);
+
+        org.coner.api.entity.HandicapGroup responseHandicapGroup = objectMapper.readValue(
```

Comment:
I see what you are saying.  The mock is never called since we are now returning a 201 with a URI in the Location header.  Removing for the other Resources as well.